cmd/main.go

```diff
@@ -221,44 +222,22 @@ func initMC() {
 
 }
 
-func installAutoCompletion(ctx *cli.Context) {
-	if ctx.Bool("no-autocompletion") || ctx.GlobalBool("no-autocompletion") {
-		return
-	}
-
-	if globalQuiet || globalJSON || !terminal.IsTerminal(int(os.Stdout.Fd())) {
-		return
-	}
-
+func installAutoCompletion() {
 	if runtime.GOOS == "windows" {
+		console.Infoln("autocompletion feature is not available for this operating system")
 		return
 	}
 
-	if completeinstall.IsInstalled("mc") {
+	if completeinstall.IsInstalled(filepath.Base(os.Args[0])) || completeinstall.IsInstalled("mc") {
```

Comment:
The second completeinstall.IsInstalled("mc") doesn't seem to be useful here.. completion won't work anyway if the binary name is different to 'mc'

Reply:
It will work it's already fixed.